packages/modal/README.md

```diff
@@ -56,13 +56,14 @@ function ExampleComponent() {
-| `setOpen`     | `function`                  | Callback to set open state of Modal component. `setOpen` accepts a boolean value, which will determine the open state of `Modal` component. | `() => {}`   |
-| `size`        | `small`, `default`, `large` | Determines `Modal` size.                                                                                                                    | `default`    |
-| `shouldClose` | `function`                  | Callback to determine whether or not Modal should close when user tries to close it.                                                        | `() => true` |
-| `children`    | `node`                      | Children that will be rendered inside `<Modal />` component.                                                                                |              |
-| `className`   | `string`                    | Style to be applied to the overlay container's root node.                                                                                   |              |
+| Prop               | Type                        | Description                                                                                                                                 | Default      |
+| ------------------ | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| `open`             | `boolean`                   | Determines open state of `Modal` component                                                                                                  | `false`      |
+| `setOpen`          | `function`                  | Callback to set open state of Modal component. `setOpen` accepts a boolean value, which will determine the open state of `Modal` component. | `() => {}`   |
+| `size`             | `small`, `default`, `large` | Determines `Modal` size.                                                                                                                    | `default`    |
+| `shouldClose`      | `function`                  | Callback to determine whether or not Modal should close when user tries to close it.                                                        | `() => true` |
+| `children`         | `node`                      | Children that will be rendered inside `<Modal />` component.                                                                                |              |
+| `className`        | `string`                    | Style to be applied to the container's root node.                                                                                           |              |
+| `contentClassName` | `string`                    | Style to be applied to the overlay div node                                                                                                 |              |
```

Comment:
People may think "overlay" refers to the dark background. I might suggest "dialog card div" or "content div" instead.

Reply:
👍